src/BannerNewsletter/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import { from, headline } from '@guardian/source-foundations';
+import { Button, SvgCross } from '@guardian/source-react-components';
+import { useEscapeShortcut, OnCloseClick, CLOSE_BUTTON_ID } from '../bannerCommon/bannerActions';
+import { NewsletterSubscribeCallback, CTA, NewsletterFrequency } from '../newsletterCommon';
```

Comment:
Is it work splitting out the type and non-type imports here?

Reply:
Mixing type declarations in with other imports doesn't seem to choke the TS engine or the bundler - maybe we can discuss coding preferences here? If we go for separation then I'll need to go through all file imports to update as this sort of mixing isn't handled by any linter warning.

Reply:
Ah okay, I thought the linter reported these for this project. I must be mixing up with other projects. Agreed if the linter is happy then it's fine (but yep let's discuss separately, it'd be nice to have consistent linting rules across projects).